CONTRIBUTING.md

```diff
@@ -1,44 +1,26 @@
-![Toolbox logo](data/logo/toolbox-logo-landscape.svg)
+![Toolbox logo](data/gfx/powerup.gif)
 
 # Contributing to Toolbox
 
 Thank you for wanting to contribute to Toolbox! We greatly appreciate your
 interest!
 
-# Table of contents
-
-- [Contributing to Toolbox](#contributing-to-toolbox)
-- [Table of contents](#table-of-contents)
-- [Reporting Bugs](#reporting-bugs)
-  - [Before Submiting a Bug Report](#before-submiting-a-bug-report)
-  - [Writing a Bug Report](#writing-a-bug-report)
-- [Making Suggestions](#making-suggestions)
-  - [Before Submitting a Suggestion](#before-submitting-a-suggestion)
-  - [Writing a Suggestion](#writing-a-suggestion)
-- [First Contribution](#first-contribution)
-- [Pull Requests](#pull-requests)
-  - [Creating a Pull Request](#creating-a-pull-request)
-  - [After Creating a Pull Request](#after-creating-a-pull-request)
-- [Little Style Guide](#little-style-guide)
-
 # Reporting Bugs
 
-## Before Submiting a Bug Report
+## Before Submitting a Bug Report
```

Comment:
I'd suggest using the word *issue* instead of *bug report* because that's the terminology GitHub uses.

Reply:
Sure.